xml/QS_system_requirements.xml

```diff
@@ -17,7 +17,127 @@
   <title>&productname; System Requirements</title>
  </info>
  <para>
-  The following system requirements apply for all types of nodes provided by
-  &productname;:
+  Before you begin the installation, please make sure your system meets all the
+  requirements listed below.
  </para>
+ <sect1 xml:id="sec.caasp.quickstart.clusreqs">
+    <title>Cluster Size Requirements</title>
+    <para>
+     &productname; only functions in a clustered configuration. It requires a 
+     group of at least four physical or virtual machines. This will yield a 
+     cluster with one &admin_node;, one &master_node;, and two &worker_node;s.
+    </para>
+    <para>
+     All clusters must contain a single &admin_node; and at least one &master_node;
+     and one &worker_node;.
+    </para>
+    <para>
+     For improved performance and reliability, an odd number of &master_node;s 
+     and any number of &worker_node;s up may be added.
+    </para>
+    <note>
+     <title>Test and Proof-of-Concept Clusters</title>
+     <para>
+      It is possible to provision a three-node cluster with only a single
+      &worker_node;, but this is not a supported configuration for deployment.
+     </para>
+    </note>
+   </sect1>
+   <sect1 xml:id="sec.caasp.quickstart.hwreqs">
+    <title>Minimum Hardware Requirements</title>
+    <itemizedlist mark="bullet" spacing="normal">
```

Comment:
I know it says "Minimum hardware requirements", but I would strongly suggest to add a disclaimer stating that these minimums are only useful for a test deployment. Some additional advise/formula on how to calculate for a real-life deplyoment would be good.

Reply:
Noted. The team are working on defining this on the internal mailing list but they do not seem to have anything firm yet.